next-app/app/page.js

```diff
@@ -138,41 +197,94 @@ function WeatherData({ apiData }) {
+                            <div className="float-end">
+                              <b className="text-end">{value.humidity}%</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Wind:</div>
+                            <div className="float-end">
+                              <b>{value.wind} mph</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Rain Chance: </div>
+                            <div className="float-end">
+                              <b>{value.precipitation}%</b>
+                            </div>
+                          </ListGroup.Item>
+                          <ListGroup.Item className="card-text">
+                            <div className="float-start">Forecast: </div>
+                            <div className="float-end">
+                              <b>{value.weathercode}</b>
+                            </div>
+                          </ListGroup.Item>
+                        </ListGroup>
                       </div>
                     </div>
                     <div className="card-footer text-center">
                       <div className="row justify-content-center">
-                        <div className="col-sm">
-                          <p className="card-text pb-5">
+                        <div className="col-sm pb-5">
+                          <p className="card-text">
                             Judgement: <b>{value.judgement}</b>
                           </p>
+                          <Button type="button" onClick={schedDate}>
+                            Schedule Date
+                          </Button>
+                          <ToastContainer className="position-fixed p-3" position="bottom-end">
+                            <Toast onClose={() => setShow(false)} show={show} delay={6000} autohide>
+                              <Toast.Header>
+                                <strong className="me-auto">
+                                  {dateRequested === 'success'
+                                    ? 'Date scheduled'
+                                    : 'Date already scheduled'}
+                                </strong>
+                              </Toast.Header>
+                              <Toast.Body>
+                                {dateRequested === 'success' ? (
+                                  <p>
+                                    The date &apos;<b>{apiDate}</b>&apos; was added to the schedule!
+                                  </p>
+                                ) : (
+                                  <p>
+                                    <b>{apiDate}</b> is non-unique! Choose a different date.
```

Comment:
I think we can reword this to something along the lines of "this date is already scheduled"